src/index.html

```diff
@@ -6,9 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu__head">
+      <div class="logo">
+        <a href="http://localhost:8080/">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="menu__nav">
+        <ul class="menu__ul">
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Apple</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Samsung</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Smartphones</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Laptops & Computers</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Gadgets</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Tablets</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Photo</a>
+          </li>
+          <li class="menu__li">
+            <a href="#" class="menu__a is-active">Video</a>
+          </li>
```

Comment:
there shoud be only 1 active tab and the underline should always be present, not only on press